hack/flakes/flake.nix

```diff
@@ -2,7 +2,7 @@
   description = "Useful flakes for golang and Kubernetes projects";
 
   inputs = {
-    nixpkgs.url = "github:NixOS/nixpkgs/nixos-unstable";
+    nixpkgs.url = "github:NixOS/nixpkgs/nixpkgs-unstable";
```

Comment:
What is this?

Reply:
This is just changing to a different nix channel, using nixpkgs rather than nixos (which is more used for the OS). It doesn't make any difference, just feels a bit more correct.